test/math.jl

```diff
@@ -661,3 +661,15 @@ end
     @test sincos(big(1)) == (sin(big(1)), cos(big(1)))
     @test sincos(big(1.0)) == (sin(big(1.0)), cos(big(1.0)))
 end
+
+@testset "rem_pio2_kernel" begin
+    # test worst case
+    setprecision(BigFloat, 4096)
```

Comment:
should the previous value of this be saved and restored so it doesn't influence later tests that run in the same worker?